docs/usermanual.md

```diff
@@ -843,6 +842,30 @@ the example case the command will be:
 }
 ```
 
+The agent provides additional information, hereafter referred as **context**, in order to be used to evaluate the
```

Comment:
```suggestion
The agent provides additional information, hereafter referred as **JEXL context**, in order to be used to evaluate the
```

To be clear it is not the sames the context information managed by ContextBroker, sometimes referred as **NGSI context**.